cluster/gce/gci/configure-helper.sh

```diff
@@ -2348,10 +2348,18 @@ EOF
     base_eventer_memory="190Mi"
     base_metrics_cpu="${HEAPSTER_GCP_BASE_CPU:-80m}"
     nanny_memory="90Mi"
-    local -r metrics_memory_per_node="${HEAPSTER_GCP_MEMORY_PER_NODE:-4}"
+    local heapster_min_cluster_size="16"
+    local metrics_memory_per_node="${HEAPSTER_GCP_MEMORY_PER_NODE:-4}"
     local -r metrics_cpu_per_node="${HEAPSTER_GCP_CPU_PER_NODE:-0.5}"
     local -r eventer_memory_per_node="500"
     local -r nanny_memory_per_node="200"
+    if [[ "${ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS:-}" == "true" ]]; then
+      # TODO: Set each of the following variables to optimized values.
+      base_metrics_memory="140Mi"
```

Comment:
You probably wanted "${HEAPSTER_GCP_BASE_MEMORY:-140Mi}". I'd drop this 'if' as it's a no-op. It's easy enough to add when you'll have the proper logic figured out.

Reply:
Good catch! Thanks. This logic has been implemented to facilitate testing of a complementary change that sets the ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS environment variable.